MapboxNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock

```diff
@@ -1,72 +0,0 @@
-PODS:
```

Comment:
It looks like this file was removed instead of being moved under MapboxCoreNavigationTests.